README.md

```diff
@@ -261,6 +261,13 @@ policies:
   regex_pattern: "\d"
 ```
 
+- `DeleteLeastRecentlyUsedFiles` - delete the least recently used files until the total kept size is greater than passed value
+
+```yaml
+- rule: DeleteLeastRecentlyUsedFiles
+  keep_at_most_mb: 512
```

Comment:
I don't think we should combine LRU and sized rule here.
Almost all rules right now works on "number of files" idea - so for LRU it'd be something like
```yaml
- rule: DeleteLeastRecentlyUsedFiles
  keep: 10
```

where 10 - number of artifacts.

Imagine the case when you setup 512MB limit with regular size for each artifact for 10Mb.
But accidentally, you put some large `zip/deb/msi` in the repo, like 640Mb.
The rule with Mb will remove all other artifacts, but will keep only the last one.